src/components/registration/steps.jsx

```diff
@@ -315,6 +315,7 @@ module.exports = {
                             <div className="gender-input">
                                 <Input name="user.genderOther"
                                        type="text"
+                                       className="no-label"
```

Comment:
Rather than needing to know about this class, could `Input` just give the class to itself if there is no `label` prop?